scripts/qesap/qesap.py

```diff
@@ -114,8 +114,25 @@ def cmd_configure(configure_data, base_project, dryrun):
     if not config.validate_ansible_config(None):
         return Status("Problems in the ansible part of the configuration")
 
-    hanamedia_content = {'hana_urls': configure_data['ansible']['hana_urls']}
-    log.debug("Hana media %s:\n%s", cfg_paths['hana_media_file'], hanamedia_content)
+    hanamedia_content = {}
+    if configure_data['apiver'] < 3:
+        match = re.search(r'http.*://(?P<ACCOUNT>.*)\.blob\.core\.windows\.net/(?P<CONTAINER>.*)/(?P<EXE>.*)', configure_data['ansible']['hana_urls'][0])
```

Comment:
With those `.*` up there, the regular expression can match to empty strings. I suggest changing those to `.+` or verifying that there's something in `match.group('ACCOUNT')`, `match.group('CONTAINER')` and `match.group('EXE')`

I also don't like that this RE limits us to Azure blobs, but let's cross that bridge when we need to do so.

Reply:
The purpose of this regexp is to support any existing apiver:2 existing config.yaml files and hopefully we could drop it in the next future. It supposes that the apiver:2 config.yaml has functioning urls in it.
I can and I will apply the change, the result will be to let the script to fails explicitly with error like "Invalid URL", that will tell the user that he/she is using not only an old apiver:2 file, but also once with bad and invalid urls :-)

Reply:
> It supposes that the apiver:2 config.yaml has functioning urls in it.

Exactly my point. That regexp would match something like `http://.blob.core.windows.net//` which is clearly a malformed URL. `.+` instead of `.*` will prevent that. (TBH, I would go stricter; something like `[^\.]+` for ACCOUNT and `[^\/]+` for CONTAINER and EXE, but I may be going overboard with it)